docs/framework/wcf/feature-details/how-to-install-and-configure-wcf-activation-components.md

```diff
@@ -45,7 +45,7 @@ This topic describes the steps required to set up Windows Process Activation Ser
     ```  
   
     > [!NOTE]
-    >  This command is a single line of text. This command enables the /\<*WCF Application*> application to be accessed using both http://localhost*/\<WCF Application>* and net.tcp://localhost/*\<WCF Application>*.  
+    >  This command is a single line of text. This command enables the /\<*WCF Application*> application to be accessed using both `http://localhost`*/\<WCF Application>* and `net.tcp://localhost/`*\<WCF Application>*.
```

Comment:
this renders a bit weird. Maybe it would be better to just have everything inside backticks?